cookietemple/lint/lint.py

```diff
@@ -14,7 +14,7 @@
 log = logging.getLogger(__name__)
 
 
-def lint_project(project_dir: str, is_create: bool = False) -> TemplateLinter:
+def lint_project(project_dir: str, skip_external: bool, is_create: bool = False) -> TemplateLinter:
```

Comment:
add a quick param explanation

Reply:
it's not like there's one for is_create :)

Reply:
Yeah haha Just saw it, sry, I should have done this before